src/main/java/uk/gov/hmcts/darts/common/entity/HiddenReasonEntity.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import org.hibernate.annotations.Immutable;
+
+@Entity
+@Table(name = "object_hidden_reason")
+@Getter
+@Immutable
+public class HiddenReasonEntity {
```

Comment:
does this not already exist as ObjectHiddenReasonEntity?

Reply:
Oh you're right it does. I'll remove my addition there.